src/Query/Builder.php

```diff
@@ -856,7 +846,7 @@ public function drop($columns)
      */
     public function newQuery()
     {
-        return new self($this->connection, $this->processor);
+        return new self($this->connection, null, $this->processor);
```

Comment:
Is a `null` second argument equivalent to constructing a new Grammar object (perhaps by way of calling `Connection::getQueryGrammar()`), as was done in the original overridden constructor (before the changes in this PR)?

Just want to confirm there's no behavioral change here.